WalletWasabi.Tests/WalletWasabi.Tests.csproj

```diff
@@ -26,6 +26,7 @@
 		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.9.0" />
 		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.10.0" />
 		<PackageReference Include="Moq" Version="4.16.1" />
+		<PackageReference Include="Shouldly" Version="4.0.3" />
```

Comment:
We are very strict with dependencies in Wasabi - we only add if it absolutely required even for Tests. Also, this would introduce a new code style. 
Can you do it without Shoudly?

Reply:
I'll delete it.